django/views/templates/technical_404.html

```diff
@@ -52,7 +52,7 @@
         {% for pattern in urlpatterns %}
           <li>
             {% for pat in pattern %}
-                {{ pat.pattern }}
+                {{ pat.pattern.regex }}
```

Comment:
Maybe the `...Pattern` could implement a `__str__` that shows the proper argument? Or an explicit method for "string in technical 404"?